tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -233,10 +261,12 @@ internal static JsonTextWriter GetJsonWriter(StringBuilder builder)
                 writer.WriteValue(_host);
             }
 
-            if (_globalTags is not null && !renderingDetails.HasRenderedTags)
+            EnrichTagsStringWithGitMetadata();
```

Comment:
Should this step be added to the `FormatCIVisibilityLog` function below too? @tonyredondo?

Reply:
That would be nice yes.

Reply:
Sure, happy to, but I'm a bit confused on what would be the best way to achieve this. Per https://github.com/DataDog/dd-trace-dotnet/pull/3652/#discussion_r1104186649 we changed things so that in CI Visibility we will use the `NullGitMetadataTagsProvider` which always returns `GitMetadata.Empty`.

I guess, for lack of a better option, `FormatCIVisibilityLog` should call ` CIEnvironmentValues Instance` and grab the `Commit` and `Repository` off of it?

Reply:
I added the support for CI Visibility in [f8d4161](https://github.com/DataDog/dd-trace-dotnet/pull/3652/commits/f8d4161a4a03ee1ec2410a476621918ab34d4cb6). So CI Visibility no longer uses the `NullGitMetadataTagsProvider`.